.gitignore

```diff
@@ -14,6 +14,7 @@ dist-ssr
 
 # Editor directories and files
 !.vscode/extensions.json
+.vscode/settings.json
```

Comment:
Could you please remove also the vscode directory? Why not `/.vscode` to have all vsc files ignored?